src/components/Settings/SettingsAddressbook.vue

```diff
@@ -160,6 +166,12 @@ export default {
 			}
 			return t('contacts', 'Copy link')
 		},
+		avatarSettingsPage() {
```

Comment:
I guess same for this, right? You did not implement a setting page yet, correct?

Reply:
Correct, only ideas so far.